### ISSUE TITLE
What are the training hyper parameters to get top1 acc>0.87? 

### ISSUE THREAD

**Issue by wjtan99** [JDAI-CV/DCL#9]
What are the training hyper parameters to get top1 acc>0.87, like batch, lr strategy? and how many epochs do you run?
I can only get 0.83 on the CUB_200_2001. I divided the data sets to 70% training, 30% validation.
Please do not close the issues so fast.
Thanks.

**Comment by Chen94yue** [comment]
Accuracy>0.87 can be achieved with default settings. Maybe, your training set is different from http://www.vision.caltech.edu/visipedia/CUB-200.html.

**Comment by wjtan99** [comment]
I am using the CUB_200_2001 data set.  But you are right, I am not using the exact train_test_split.txt.  Anyway, by adjusting starting lr, and batch size, I can now achieve top1 acc>0.87. 
Thanks any way.

**Comment by akira-l** [comment]
Great! Let we know if anything doesn’t work :)

**Comment by goldentimecoolk** [comment]
> Accuracy>0.87 can be achieved with default settings. Maybe, your training set is different from http://www.vision.caltech.edu/visipedia/CUB-200.html.

Hi guys, do you achieve the same performance on CUB_200_2011 dataset?

**Comment by goldentimecoolk** [comment]
> I am using the CUB_200_2001 data set. But you are right, I am not using the exact train_test_split.txt. Anyway, by adjusting starting lr, and batch size, I can now achieve top1 acc>0.87.
> Thanks any way.

Hi guys, do you achieve the same performance on CUB_200_2011 dataset?

**Comment by MichaelLiang12** [comment]
I retrained the model with default setting. My best result is 87.30% on CUB.

**Comment by sanduanji** [comment]
i can just get 85.95% on CUB 200-2011 dataset with default hyper parameters. Is there anything wrong?

**Comment by tinyboy28** [comment]
> I am using the CUB_200_2001 data set. But you are right, I am not using the exact train_test_split.txt. Anyway, by adjusting starting lr, and batch size, I can now achieve top1 acc>0.87.
> Thanks any way.

Hello!, thanks for your sharing, I'm wondering that how did you adjust the parameters that made you achieve top1>0.87? I'm using the original CUB200 dataset split(train : test=1 : 1), train batchsize=16, train epoch=180, start_lr=0.0008, but I can only get top1=0.86, can you share some experience about your parameters adjusting?
